content/Hardware/Portenta Family/About-ADC-reference-voltage-on-Portenta-H7.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "About ADC reference voltage on Portenta H7"
+---
+
+The **analog-to-digital converter** (ADC) on Portenta H7 switches between internal and external reference voltage automatically, so unlike some other boards, [analogReference()](https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/) should **not** be used.
+
+<a id="internal-reference"></a>
+
+## Using the internal reference voltage (3.1 V)
```

Comment:
Explain what is the purpose of normalizing? why would we need to do that?

Reply:
It's an alternative to using the AREF pin. Instead of applying an external reference voltage we use the 3.1 V and normalize so that 2.0 V corresponds to max (1.0). It will be noiser but can be done only using code.